Gemfile

```diff
@@ -15,7 +17,7 @@ gem 'sidekiq-scheduler', '~> 2.0' # TODO: explanation
 
 gem 'aasm'
 gem 'attr_encrypted'
-gem 'aws-sdk'
+gem 'aws-sdk', '~> 3'
```

Comment:
> Since version 3 is backwards compatible with version 2, you should not need to make additional changes beyond those described below.

Reply:
Interestingly there was a minor update required to some shrine tests - just needed to provide a dummy access key id and secret key in the test config. 

And context is that signed URL generation is different between aws-sdk 2 and 3.